### DISCUSSION TITLE
Implementing Culture Change to advance Green Software

### DISCUSSION THREAD

**Discussion by cerianmorgan** [Green-Software-Foundation/discussions#42] [Ask Anything]
Hi there, I’m Cerian Morgan (she/her). I work at HSBC in people strategy. I’m keen to connect with anyone who is in the midst of or who has implemented any culture change on this topic or embedded any learning / behaviour change on green software practices. Thanks!
Cerian 
https://www.linkedin.com/in/cerianmorgan

**Comment by NAMRATA-WOKE** [comment]
Hi @cerianmorgan, nice to meet you! We've got a few folks in our community who've taken lead on internal advocacy to support a sustainability shift within their organizations and center green software in their dev strategy. Have you signed up to participate in upcoming Community Working Group meetings and/or interested in engaging in this discussion more broadly? 

You might want to connect with @adamj89 who heads our community efforts. I'll share this note with others in our community who are interested in connecting with hopes of facilitating some introductions for you.
